src/apollo/schema.graphql

```diff
@@ -49,3 +50,8 @@ union UpdateRunDetailsResponse = UpdateRunDetailsSuccess | UpdateRunDetailsFailu
 type UpdateRunDetailsSuccess {
   run: Run!
 }
+
+type Version {
+  installed: String!
```

Comment:
naming this as `Current` might be more relevant as I understand there can be multiple versions of a package installed in a python environment ( @limdauto @AntonyMilneQB can correct me if I'm wrong)

Reply:
there can't be multiple versions of a package installed in the same python environment. There can be many environments.

Reply:
Thanks for clarifying @limdauto - on that note, I've heard that it is possible to install multiple versions of a package by specifying it to be installed in a different directory other than the default `site-packages`? ( not that it is a common use case at all) Again, I'm no python dev, so you know best 😄 

On the subject, `Current` version to me sounds more intuitive as we normally refer to 'current' and 'latest' versions - curious to hear your thoughts.

Reply:
The reason I didn't like `current` is because I didn't know what it was describing, and it seemed like it could describe both. As in: is it the _current_ version the user has installed? Or is it the _current_ version that's released on PyPI? 

With `installed` and `latest` I think it's more clear. It might even be clearer if it's `installed` and `latestRelease`.

Reply:
I also don't like `current` for the same reason as @tynandebold mentioned. `installed` and `latest` make most sense to me.

With that said, we are already using `current` as the variable name [where we do the CLI nudge](https://github.com/kedro-org/kedro-viz/blob/main/package/kedro_viz/launchers/cli.py#L76-L80). So whatever we go for, let's be consistent so we don't have current, latest and installed in the codebase.

Reply:
@studioswong you could indeed install different version of the same Python package to different locations. But when we do `from kedro_viz import __version__`, it will be well-defined and import from only one installation of the package. So it does make sense to refer to the "installed Python package" as a unique entity.